metaci/release/tests/test_utils.py

```diff
@@ -57,4 +57,52 @@ def test_send_release_webhook__error(mocked_responses, mocker, transactional_db)
     with pytest.raises(
         Exception, match="ImplementationStep matching query does not exist."
     ):
-        send_release_webhook(project_config, release, "INFRA.instance1")
+        send_release_webhook(release, "INFRA.instance1")
+
+
+def test_send_submit_webhook(mocked_responses, mocker, transactional_db):
+    mocker.patch(
+        "metaci.release.utils.settings",
+        METACI_RELEASE_WEBHOOK_URL="https://webhook",
+        METACI_CHANGE_CASE_URL_TEMPLATE="{case_id}",
+        METACI_RELEASE_WEBHOOK_ISSUER="MetaCI",
+        METACI_RELEASE_WEBHOOK_AUTH_KEY="test",
+    )
+    mocked_responses.add(
+        "POST",
+        "https://webhook/case/None/submit",
+        json={"success": True, "id": "2"},
+    )
+
+    project_config = Mock(project__package__name="Test Package")
+    project_config.get_version_for_tag.return_value = "1.0"
+    release = ReleaseFactory()
+    send_submit_webhook(release, "INFRA.instance1")
+    assert not release.change_case_link
+
+
+def test_send_submit_webhook__disabled(mocked_responses):
+    send_submit_webhook(None, None)
+    assert len(mocked_responses.calls) == 0
+
+
+def test_send_submit_webhook__error(mocked_responses, mocker, transactional_db):
+    mocker.patch(
+        "metaci.release.utils.settings",
+        METACI_RELEASE_WEBHOOK_URL="https://webhook",
+        METACI_RELEASE_WEBHOOK_ISSUER="MetaCI",
+        METACI_RELEASE_WEBHOOK_AUTH_KEY="test",
+    )
+    mocked_responses.add(
+        "POST",
+        "https://webhook/case/None/submit",
+        json={
+            "success": False,
+            "errors": [
+                {"message": "ImplementationStep matching query does not exist."}
+            ],
+        },
```

Comment:
Based on what I see in gus-bus (https://github.com/SFDO-Tooling/gus-bus/blob/main/gus_bus/main.py#L76-L79), the expected response here would be:

```suggestion
        json={
            "success": False,
            "errors": ["Error submitting change case."],
        },
```